src/pages/personalisation/Intro.js

```diff
@@ -0,0 +1,85 @@
+/* @flow */
+
+"use strict";
+
+import React from 'react';
+import Router from "react-router";
+import mui from "material-ui";
+import reactMixin from "react-mixin";
+
+import Personalisation from '../../mixins/Personalisation';
+import components from '../../components';
+import icons from '../../icons';
+import * as iss from '../../iss';
+
+/*::`*/@reactMixin.decorate(Router.Navigation)/*::`;*/
+/*::`*/@reactMixin.decorate(Router.State)/*::`;*/
+/*::`*/@reactMixin.decorate(Personalisation)/*::`;*/
+class Intro extends React.Component {
+    constructor(props: Object) {
+        super(props);
+        this.state = {
+        };
+    }
+
+    static getSearch(request: iss.searchRequest): ?iss.searchRequest {
+        return request;
+    }
+
+    // flow:disable
+    get seekingHelpWith(): string {
+        try {
+            var category = this.context.controller.category;
+            return category.byline.toLocaleLowerCase() ||
+                `with ${category.name.toLocaleLowerCase()}`;
+        } catch (e) {
+            var search = this.context.controller.props.params.search;
```

Comment:
This is a language style thing I guess. Python would say try and catch an exception. Flexible?